requirements-test.txt

```diff
@@ -7,4 +7,5 @@ pytest-cov==2.5.1
 pytest-parallel==0.0.2
 pymongo==3.7.1
 pytest-mongodb==2.1.2
-black==18.6b3
\ No newline at end of file
+black==18.6b3
+scipy==1.1.0
```

Comment:
scipy is not a test requirment, it should go into `install_requires` of `setup.py` FIY

Reply:
done.